lib/modules/manager/helmv3/artifacts.ts

```diff
@@ -32,20 +35,12 @@ async function helmCommands(
   repositories: Repository[]
 ): Promise<void> {
   const cmd: string[] = [];
-  // set cache and config files to a path in privateCacheDir to prevent file and credential leakage
-  const helmConfigParameters = [
-    `--registry-config ${upath.join(privateCacheDir(), 'registry.json')}`,
-    `--repository-config ${upath.join(privateCacheDir(), 'repositories.yaml')}`,
-    `--repository-cache ${upath.join(privateCacheDir(), 'repositories')}`,
-  ];
-
   // get OCI registries and detect host rules
-  const registries: RepositoryRule[] = repositories
+  const ociRepositoryRules: OCIRepositoryRule[] = repositories
     .filter(isOCIRegistry)
     .map((value) => {
       return {
-        ...value,
-        repository: value.repository.replace('oci://', ''),
+        host: value.repository.replace('oci://', ''),
```

Comment:
```suggestion
        ...value,
        repository: value.repository.replace('oci://', ''),
```
You do not have to create your own type.